packages/view/backend/sirius-components-view-diagram-edit/src/main/resources/plugin.properties

```diff
@@ -130,6 +130,7 @@ _UI_BorderStyle_borderSize_feature=Border Size
 _UI_BorderStyle_borderLineStyle_feature=Border Line Style
 _UI_InsideLabelStyle_withHeader_feature=With Header
 _UI_InsideLabelStyle_displayHeaderSeparator_feature=Display Header Separator
```

Comment:
I think we should be more precise here because it might not be obvious in the property view what these option do.
Maybe something like : 

- Display Header Separator with children
- Always Display Header Separator

Or add a hint in the property view that could give more information

Reply:
I change the property label for `AlwaysShowSeparator` to be more explicit.
It would have been great to add a hint but currently, to provide an helper text to a details form widget, we have to redefine a page custom which means to declare all the other widget manually

Reply:
Finally I changed to used an enum (NEVER, ALWAYS, IF_CHILDREN) which makes it much clearer for the users